video/out/cocoa_common.m

```diff
@@ -873,4 +883,16 @@ - (void)windowDidBecomeKey:(NSNotification *)notification
     [self didChangeMousePosition];
 }
 
+- (void)windowDidMiniaturize:(NSNotification *)notification
+{
+    struct vo_cocoa_state *s = self.vout->cocoa;
+    s->pending_events |= VO_EVENT_WIN_STATE;
+}
+
+- (void)windowDidDeminiaturize:(NSNotification *)notification
```

Comment:
Is this method necessary? It's the exact same as `windowDidMiniaturize`, and it looks like you don't even call it from the window delegate.